### ISSUE TITLE
2.x No Longer Uses Device Nickname

### ISSUE THREAD

**Issue by michaelcresswell** [jellyfin-archive/jellyfin-roku-legacy#1666]
## Software Versions

- Jellyfin Server Version: 10.8.13
- Roku Client Version: 2.0.2

## Describe the bug
Jellyfin 2.x no longer reports device nickname to the server

## How To Reproduce
1. Connect to a Jellyfin server with the Roku client
2. Go to server Administration -> Dashboard
3. Click Devices
4. Observe the Roku device

## Expected behavior
The device nickname is part of the device identifier

## Screenshots
From the Admin Page:

2.0.2
![image](https://github.com/jellyfin/jellyfin-roku/assets/3800846/05bbc790-05d2-48bd-82cb-0f0e9fbccc1e)
G000X seems to correspond to the Hardware ID of G109X

1.6.6
![image](https://github.com/jellyfin/jellyfin-roku/assets/3800846/865b8bba-7ded-46e0-80bf-eb90584e6ecd)
Living Room TV (abbreviated in the screenshot) is the nickname shown on the Roku app

## Connection Information

- Is server local or remote? Local
- Is server connection HTTP or HTTPS?
  Jellyfin <— HTTP —> Reverse Proxy <— HTTPS —> Client

## Additional context
The 1.6.6 client reports the nickname which makes it easier to indentify clients, especially when multiple clients have similar hardware.

It’s a minor regression but something I noticed in the dashboard.

**Comment by krkc** [comment]
This looks like the commit where the change was made:

https://github.com/jellyfin/jellyfin-roku/commit/d0bbd696f60089ec00fb2dd4eeb0d18285acd284

**Comment by michaelcresswell** [comment]
Interesting…feel free to close this as not a bug if that was intended, @cewert

**Comment by cewert** [comment]
@michaelcresswell yea the change was on purpose but I don't use a Roku TV. It appears my changes are only an improvement for roku devices and not roku TVs. I'll have to look into updating the auth header based on device type.

This is how the changes look for me:

![roku-jellyfin-dash-change](https://github.com/jellyfin/jellyfin-roku/assets/16514652/331ed182-ac25-4370-ad2f-eafcb854e6c5)

**Comment by michaelcresswell** [comment]
Do you have nicknames setup for your devices? Just curious if that 1.6.6 valve was some auto-generated default name or if non-TVs are really that different.

(https://my.roku.com/dashboard -> My Devices -> Nickname; couldn’t find a way to do it locally)

**Comment by cewert** [comment]
> Do you have nicknames setup for your devices?

I don't think so but this roku was setup using someone else's email so I could be wrong.
